### ISSUE TITLE
BitDefender detects Rotki as ransomware

### ISSUE THREAD

**Issue by drgonzo31** [rotki/rotki#1819]
## Problem Definition
Running Rotki on a windows platform that is using BitDefender Antivirus. Rotki gets detected as ransomeware and is blocked from running.  Workaround is to add exception rules for BitDefender.

## Logs
"..rotki.exe manifests ransomware behavior and was blocked. Several files were encrypted but we successfully restored all of them. You can find the restored files list below"
Restored Files:
C:\ProgramData\NVIDIA Corporation\Drs\nvdrssel.bin

The rotkenlchen.log file is empty.  

### System Description

Operating system: Windows 10 Home,
Rotki version: rotki-win32-v1.9.0.

**Comment by LefterisJP** [comment]
Hey @drgonzo31 thanks a lot for the issue and letting us know of this funny behavior with Bitdefender.

There is not much we can do about it since this is very specific to your setup and another piece of software over which we have no control over.

Naturally Rotki does not contain any viruses, its code is open and used by many people around the world.

@kelsos is that one of those things that would go away with signing the binary for Windows?
https://github.com/rotki/rotki/issues/1224

Even though this Bitdefender thing is not a Microsoft product.

**Comment by kelsos** [comment]
@drgonzo31 just to confirm rotki was downloaded from our GitHub releases and the checksum verifies? I will check the binary with bitdefender on a windows VM and if there are issues I will submit the file for review as a false positive.

**Comment by drgonzo31** [comment]
Hello Kelsos, 
Downloaded rom  github.
SHA512 hash of rotki-win32-v1.9.0.exe:
2871a3431df889f509ee0f6106b7e35317f90edc3146f1e47580eaededb4a0e72a232290bd5937b20fbf31ea6b2394aa50f2b403f8bdd018e8d3c6ef33b87c25

**Comment by kelsos** [comment]
OK, the hash matches. That is really weird. Was this done straight from the start or did it take some time?

I installed Bitdefender Total Security (is that the correct product?) and activated a trial on a virtual machine and so far there have been no errors.

**Comment by LefterisJP** [comment]
@kelsos So can you post your findings here on why it's a false positive? Also did you make a report with them about it? Asking to see what is needed to close this issue.

**Comment by kelsos** [comment]
So TLDR: This is a false positive that happens under very specific conditions. 
The condition seems to be that the Nvidia driver has to also be installed.

I did some investigation by installing BitDefender on a fresh virtual machine running Windows 10 Pro. I downloaded the rotki v1.9.0 executable from GitHub and run the rotki. Just to be sure I also restored my database from the premium service.

After letting it run for a few hours I got nothing. Everything worked fine, no warnings, no errors. Scanning reported the file clean.

In the meanwhile, I also searched for the specific GPU driver file being restored by BitDefender.
After a quick search about `C:\ProgramData\NVIDIA Corporation\Drs\nvdrssel.bin BitDefender` on google you can has a few results of different software getting exactly the same error.

#### Examples:
https://steamcommunity.com/app/718850/discussions/0/1643170269586608158
https://forums.launchbox-app.com/topic/56199-latest-bitdefender-update-gives-the-following/
https://www.reddit.com/r/qBittorrent/comments/fodqmp/qbittorent_has_ransomware_on_it_now_dont_use/

This seems to be a common false-positive reported by BitDefender.

**Comment by drgonzo31** [comment]
Hello again,

Sorry I didn't back sooner I've been busy. It was Bitdefender antivirus. It didn't do it straight away. It took sometime. 

I thought it was a false positive also, I wasn't concerned about being legitimate ransomeware.  Just letting you know incase others had same problem.

Regards

Dylan



Thanks for
